bin/console

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'bundler/setup'
+require 'bambora'
+
+# You can add fixtures and/or initialization code here to make experimenting
+# with your gem easier. You can also use a different console, if you like.
+
+# (If you use this, don't forget to add pry to your Gemfile!)
+require "pry"
```

Comment:
Do you need to add `pry` to the `Gemfile`?

Reply:
Oof. Got confused between branches. My next pr includes pry, but I'll add it here.

Reply:
Should this be in single quotes?